scripts/upgrade.sh

```diff
@@ -6,13 +6,13 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
 $DIR/validate.sh;
 
 if [ -z "$1" ]; then
-    $(npm bin)/npm-check-updates --prod --upgrade
+    npx npm-check-updates --prod --upgrade
```

Comment:
My only concern is, doesn't this make this repo a little less usable out of the box? Need to have npx installed as a global for this to work, right?

Or can we alias `npx` to `$(npm bin)/npx`?

Reply:
The `npx` binary actually part of `npm` now. It was [introduced back in version 5.2.0](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b) to make it easier to use cli tools. My thinking here is that `npx` can be used for running anything out of `node_modules/.bin`.

Reply:
oh sweet! didn't realize that.